ci/build-in-docker.sh

```diff
@@ -31,7 +31,7 @@ case "$ARCH" in
 esac
 
 # libassuan-static is supported only from 3.19 onwards
-image="$image_prefix"/alpine:3.19
+image="$image_prefix"/alpine:latest
```

Comment:
We want to use a specific version.

Reply:
`image_prefix` likely isn't needed any more, specifying the platform is sufficient nowadays and has been for a few years. Please try removing that variable altogether.

Reply:
Indeed. 👍 
I don't know why it fails for ARM now.